Packs/Oletools/Scripts/OletoolsScript/OletoolsScript.py

```diff
@@ -187,8 +187,8 @@ def main():  # pragma: no cover
     ole_command = args.get('ole_command')
     attach_id = args.get('entryID', '')
     file_info = demisto.getFilePath(attach_id)
-    show_decoded = args.get('decode', False)
-    password = argToList(args.get('password', ''))
+    show_decoded = argToBoolean(args.get('decode', False))
+    password = args.get('password', '')
```

Comment:
From your comment to the last review, shouldn't it stay with `argToList` and mark it as an array in the YML file?

Reply:
@tomer-pan said he prefers this argument not as a list. so i will get it as a single password and then rap it up in the code